package.json

```diff
@@ -43,6 +43,7 @@
     "pre-push": "yarn lint"
   },
   "resolutions": {
+    "@metamask/snaps-sdk@6.1.0/@metamask/providers": "17.1.1",
```

Comment:
- Fixes build error due to package version mismatch: [MetaMask/core/actions/runs/10011688901/job/27675682526?pr=3645](https://github.com/MetaMask/core/actions/runs/10011688901/job/27675682526?pr=3645)
- Removal is blocked by a `@metamask/snaps-sdk` release with `@metamask/providers` bumped to `>=17.1.1`.
  - See: [Fix regressions introduced by @metamask/providers@17.1.1](https://github.com/MetaMask/snaps/pull/2579)